docs/user/aws/install.md

```diff
@@ -60,5 +63,110 @@ The OpenShift console is available via the kubeadmin login provided by the insta
 
 ![OpenShift web console](images/install_console.png)
 
+## Example: User Provided Infrastructure (UPI)
```

Comment:
I feel like UPI should be split out into a separate file and linked from here.  We want to make it easy to find, but also easy to skip over.  The IPI docs are short enough that I'm fine keeping them in this file, although I'm also ok splitting them out if you feel it would be tedious for UPI users to scroll past the IPI docs.

Reply:
Fixed.